### PR TITLE
Added IN_VOTER test case

### LINKED ISSUE #22: Detect IN_VOTER
### Description
Each Indian citizen is issued a unique voter ID by the Indian Election Commission once they reach 18 years of age. It serves as identity proof for citizens casting their ballot in municipal, state, and national elections.

### Detection Steps 
Each ID is 10 characters, consisting of 3 letters followed by 7 digits.

### PR DESCRIPTION
**Pull request by danabrunson**

### CODE REVIEW

**Review by cgboyce — APPROVED** [review]

File: test_team_tech_baddies.py [hunk 1]
```diff
@@ -64,7 +64,27 @@ def test_in_vehicle_registration(self):
...
+        for b in begin:
+            for e in ending:
+                voter = ''.join([b,e])
+                # Check context score should be 0.75
+                result = analyze_text('My Voter ID is ' + voter, ['IN_VOTER'])
+                print(result)
+                self.assertEqual('IN_VOTER', result[0].entity_type)
+                self.assertEqual(0.75, result[0].score)
+
+                # Check no context
```
Comment (claesmk) [inline test_team_tech_baddies.py:96]: voter is a context word for IN_VOTER according to the [opensource](https://github.com/microsoft/presidio/blob/cc31bb61982311d7e2ad7e3b5334da797cdb6f98/presidio-analyzer/presidio_analyzer/predefined_recognizers/in_voter_recognizer.py#L37) - so I don't see how this is a check for no context as stated in the comment.

**Review by claesmk — CHANGES_REQUESTED** [review]
I would suggest improving the no context case as noted.

**Review by claesmk — APPROVED** [review]
You missed what I was saying about context words.  I made the change for you

**Comment by claesmk** [comment]
Is this a duplicate of #48 ? which one is correct?

**Pull request merged**